docs/.vuepress/components/DoDont.vue

```diff
@@ -35,32 +35,11 @@ export default {
       type: Array,
     }
   },
-  beforeMount() {
-    this.examples.forEach((example, idx) => {
-      this.$options.components[`comp-${example.type}${idx}`] = { ...Vue.compile(`<span>${example.caption}</span>`) };
-    })
-  },
   methods: {
-    altText(ex) {
-      if (ex.alt) return ex.alt;
-      else if (!ex.alt && ex.type === 'do') return 'Do example image';
```

Comment:
this alt text is not helpful, if someone forgot to add alt to an example it would be better to just show `alt=""` so that the image is ignored by the screenreader and the actual do/don't/caution text is read